assets/scss/common/_focus-tooltip.scss

```diff
@@ -6,14 +6,14 @@
             top: 50%;
             border-width: remCalc(10px);
             border-style: solid;
-            border-color: transparent transparent $adminBar-tooltip-bg-backgroundColor transparent;
+            border-color: transparent transparent $tooltip-backgroundColor-focus transparent;
         }
 
         &:after {
             content: attr($attr);
             padding: remCalc(4px) remCalc(6px);
-            background-color: $adminBar-tooltip-bg-backgroundColor;
-            color: white;
+            background-color: $tooltip-backgroundColor-focus;
+            color: $tooltip-textColor-focus;
```

Comment:
Var name here also looks like with "-focus" modifier. I believe we should better name it as focusTooltip-backgroundColor and focusTooltip-textColor, based on mixin name